src/util/subprocess.h

```diff
@@ -169,7 +169,7 @@ namespace util
     //
 
     if (force == false && argument.empty() == false &&
-        argument.find_first_of(L" \t\n\v\"") == argument.npos) {
```

Comment:
In ce950ec20fa0d246d2fd44f562a864d3ff83d399 "Fix quote issue on Windows ": This isn't in a Windows block, so I'm guessing it doesn't have any effect on other OS's? Is this alos going to be upstreamed?

Reply:
Upstreamed in https://github.com/arun11299/cpp-subprocess/pull/113.